services/common/src/components/tailings/QualifiedPerson.tsx

```diff
@@ -1,242 +1,212 @@
-                                description={false}
-                            />
-                        </Col>
-
-                        <Typography.Paragraph>No Data</Typography.Paragraph>
-                    </Row>
-                )}
-                <Typography.Title level={4} className="margin-large--top">
-                    TSF Qualified Person Term
-                </Typography.Title>
-                <Row gutter={16}>
-                    <Col span={12}>
-                        <Field
-                            id="qualified_person.start_date"
-                            name="qualified_person.start_date"
-                            label="Start Date"
-                            disabled={fieldsDisabled}
-                            component={RenderDate}
-                            required={!fieldsDisabled}
-                            validate={!fieldsDisabled && [required, dateNotInFuture, validateQPStartDateOverlap]}
-                        />
-                    </Col>
-                    <Col span={12}>
-                        <Field
-                            id="qualified_person.end_date"
-                            name="qualified_person.end_date"
-                            label="End Date"
-                            disabled={fieldsDisabled}
-                            validate={!fieldsDisabled && [dateInFuture]}
-                            component={RenderDate}
-                        />
-                    </Col>
-                </Row>
+              <Empty
+                image={Empty.PRESENTED_IMAGE_SIMPLE}
+                imageStyle={{ transform: "scale(1.5)" }}
+                description={false}
+              />
             </Col>
+
+            <Typography.Paragraph>No Data</Typography.Paragraph>
+          </Row>
+        )}
+        <Typography.Title level={4} className="margin-large--top">
+          TSF Qualified Person Term
+        </Typography.Title>
+        <Row gutter={16}>
+          <Col span={12}>
+            <Field
+              id="qualified_person.start_date"
+              name="qualified_person.start_date"
+              label="Start Date"
+              disabled={fieldsDisabled}
+              component={RenderDate}
+              required={!fieldsDisabled}
+              validate={!fieldsDisabled && [required, dateNotInFuture, validateQPStartDateOverlap]}
+            />
+          </Col>
+          <Col span={12}>
+            <Field
+              id="qualified_person.end_date"
+              name="qualified_person.end_date"
+              label="End Date"
+              disabled={fieldsDisabled}
+              validate={!fieldsDisabled && [dateInFuture]}
+              component={RenderDate}
+            />
+          </Col>
         </Row>
-    );
+        <Row>
+          <Col span={24}>
+            <PartyAppointmentTable
+              canEditTSF={canEditTSFAndEditMode}
+              eor_or_qp="qualified_persons"
```

Comment:
I'd be tempted to use the MinePartyAppointmentTypeCodeEnum here! Just because it's a constant we're using elsewhere. Seems harmless.

Reply:
Unfortunately `qualified_persons` and `engineers_of_record` don't fit with that (note the plural), as they're the field name of the grouping of those items for a tsf.